plugins/maven/src/main/java/org/wildfly/swarm/plugin/maven/StartMojo.java

```diff
@@ -70,9 +70,6 @@
     @Parameter(alias = "debug", property = SwarmProperties.DEBUG_PORT)
     public Integer debugPort;
 
-    @Parameter(alias = "jvmArguments", property = "thorntail.jvmArguments")
-    public List<String> jvmArguments = new ArrayList<>();
-
```

Comment:
This shouldn't really be removed.

Reply:
Ah. I see you put it into `AbstractSwarmMojo`. I don't think that's a good idea, as it has subclasses that don't respect the `jvmArguments` setting. IMHO, it should be just copied to `MultiStartMojo`.

Reply:
Fine, I will just copy it into MultiStartMojo, makes sense.